docs/modules/operation/pages/snmp-poller/concepts.adoc

```diff
@@ -124,24 +82,40 @@ The default version of this file follows:
        <interface name="Ethernet" criteria="snmpiftype = 6" interval="300000" user-defined="false" status="on" port="161" timeout="1000" retry="1" max-vars-per-pdu="10" />
    </package-->
 </snmp-interface-poller-configuration>
-```
+----
 
-Explanation of the configuration options available:
+Configuration options available:
 
 snmp-interface-poller-configuration::
 Top-level element for the snmp-interface-poller-configuration.xml configuration file.
-Several global configuration attributes are available:
-
-* interval: default polling interval in milliseconds.
-* threads: maximum number of threads used for SNMP polling.
-* service: SNMP service string; almost always "SNMP".
-* useCriteriaFilters: Flag that indicates if interfaces that do not match the criteria attribute of an interface element in an otherwise matching package should be polled with default settings or not.
-
-When "false" such an interface will be polled with default settings; when "true" that interface will not be polled.
-* up-values: comma-separated list of valid integer values (1-7) to treat as equivalent to `up(1)` when sending status change events for an interface.
+The available attributes are:
++
+[options="header, autowidth"]
+[cols="2,3"]
+|===
+| Attribute
+| Description
+
+| interval
+| Default polling interval in milliseconds.
+
+| threads
+| Maximum number of threads used for SNMP polling.
+
+| service
+| SNMP service string.
+Almost always "SNMP".
+
+| useCriteriaFilters
+a| Flag that indicates if interfaces that do not match the criteria attribute of an interface element in an otherwise matching package should be polled with default settings or not.
```

Comment:
```suggestion
a| Flag that indicates if interfaces that do not match the criteria attribute of an interface element in an otherwise matching package should be polled with default settings.
```